src/ng/compile.js

```diff
@@ -1877,6 +1877,36 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         return value || null;
       }
 
+      function setupControllers($element, attrs, transcludeFn, controllerDirectives, isolateScope, scope) {
```

Comment:
We would also need `directive` and possibly other stuff. Let's not go there for now.